packages/utils/telemetry-utils/src/errorLogging.ts

```diff
@@ -420,4 +437,14 @@ class NormalizedExternalError extends LoggingError {
             overwriteStack(this, errorProps.stack);
         }
     }
+
+    /**
+     * Determines if a given object is an instance of a NormalizedExternalError
+     * @param object - any object
+     * @returns - true if the object is an instance of a NormalizedExternalError, false if not.
+     */
+    public static typeCheck(object: any): object is NormalizedExternalError {
+        return object.errorType !== undefined
```

Comment:
Not sure this is right. Compare with isFluidError, might be nearly equivalent. And not sure it's necessary, this class is an implementatiion detail and maybe shouldn't be reasoned over by other code